### ISSUE TITLE
Overfitting double check

### ISSUE THREAD

**Issue by cLawson101** [jessepisel/summer_fellows#156]
Redo data set to make sure that the validation set and the train/test set have no overlapping images.
